Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -134,6 +134,11 @@
     pointSize : "5"
 });
 
+addStyle('Clamp and Mix', {
+    color : "clamp(color(), 0.0, 100.0)",
+    pointSize : "mix(0.0, 2.0, 0.5) * 5"
+});
```

Comment:
Use some of the point cloud properties in here so that the point cloud is more dynamic looking. Also right now we technically don't support using clamp with anything but numbers (this will change soon).